### ISSUE TITLE
No devices found - Neural Compute Stick 2

### ISSUE THREAD

**Issue by jakubrachwalski** [acharroux/Movidius-On-MacOS#3]
I have Neural Compute Stick 2. When going through the instructions - everything works. However, when I go to the section "Run some Movidius SDK's samples" and execute the "make run" command, I receive the following error:

$ make run
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C inception_v1/. run
test -f output/inception-v1.meta || ((wget http://download.tensorflow.org/models/inception_v1_2016_08_28.tar.gz && tar zxf inception_v1_2016_08_28.tar.gz && rm inception_v1_2016_08_28.tar.gz) && ./inception-v1.py)
test -f graph || mvNCCompile -s 12 output/inception-v1.meta -in=input -on=InceptionV1/Logits/Predictions/Reshape_1
./run.py
**No devices found**
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C inception_v3/. run
test -f output/inception-v3.meta || ((wget http://download.tensorflow.org/models/inception_v3_2016_08_28.tar.gz && tar zxf inception_v3_2016_08_28.tar.gz && rm inception_v3_2016_08_28.tar.gz) && ./inception-v3.py)
test -f graph || mvNCCompile -s 12 output/inception-v3.meta -in=input -on=InceptionV3/Predictions/Reshape_1
./run.py
**No devices found**

Have you seen this issue before? DO you know what may be the issue?

**Comment by acharroux** [comment]
Hi 
Can you confirm it's second version of the NCS, the brand new one ?
I cannot get back the exact URL right now, but I remember reading Intel's doc about NCS 2, that this version cannot be used with this SDK :(

The new NCS must be used with another Intel's SDK based on openVINO. This new SDK can manage new NCS as well as older NCS but unfortunately the whole reverse is not possible : using new NCS with older SDK.

I've read somewhere that Intel is well know to make big changes in its SDK and breaking up back-compatibility. Looks like it's quite true :(
